nucypher/blockchain/economics.py

```diff
@@ -55,11 +56,12 @@ class BaseEconomics:
     nunits_per_token = 10 ** __token_decimals  # Smallest unit designation
 
     # Period Definition
-    _default_hours_per_period = 24
+    _default_hours_per_period = 24 * 7
```

Comment:
`_default_hours_per_period = 24 * os.getenv('NUCYPHER_HOURS_PER_PERIOD', 7)`

Reply:
this is not a suggestion... just a thought.

Reply:
this variable is close to constant, consider this like different eras: genesis 1 period - 1 day, current era - 1 period - 7 days.